### PR TITLE
include setup guide for AnkiDroid

### PR DESCRIPTION
**Pull request by DreamFlasher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,15 @@ and put it in `~/Anki/addons`.
...
 [addons21]: https://apps.ankiweb.net/docs/addons.html#_add_on_folders
 
+### AnkiDroid
+
+Advanced -> Custom sync server
+
+Sync url: `http://your.server:27701`
+Media sync url: `http://your.server:27701/msync`
+
+Make sure to not use https, no trailing slashes and include the port. You'll be asked to login again, the email is the username (no full email necessary).
```
Comment (AntonOfTheWoods) [inline README.md]: HTTPS works just fine. I have HTTPS set up and it works with zero issues. And the login definitely doesn't have to be your email. My server doesn't know anything about emails.

Comment (DreamFlasher) [reply README.md]: No it doesn't. It requires a reverse proxy being set up: https://github.com/tsudoko/anki-sync-server/issues/17 -- Login: Please re-read what I wrote, that's what I am pointing out :)

Comment (AntonOfTheWoods) [reply README.md]: You shouldn't be using this without a reverse proxy unless you are on a secure private network (and even then...). Saying "Make sure not to use https" suggests there will be issues if you try to use it behind a reverse proxy. You also don't have to include the port if it is the standard one for the protocol (80 for http, 443 for https), which you may or may not choose to use on your proxy.

I'm not saying that the information you are trying to get across might not be useful, just that the way you put it there makes things worse, IMHO.

**Comment by DreamFlasher** [comment]
No it simply means that you can't use it without additional effort. Feel free to create your own PR or modify this one.

**Comment by luoliyan** [comment]
I think the issue is that there are additional, non-trivial steps that need to be taken to handle an SSL connection, beyond what is covered in the installation instructions. I've had a go at setting up HAProxy to handle the SSL termination on my home network, and it's far from the most straightforward thing in the world. As a first attempt, it seems the strategy is something like:

a. Install HAProxy and figure out where the config file lives on my system
b. Read enough of the docs to create a workable `haproxy.cfg`:
 - bind the front-end to the desired interface
 - forward the unencrypted traffic to `ankisyncd`
 - stuff this up a few times because I misunderstand the syntax/protocol

c. Realise that I need an SSL certificate:
 - read the OpenSSL documentation
 - generate a `.key` file and a `.csr` file
 - create a self-signed certificate 
 - point HAProxy to the certificate

d. Realise that clients reject self-signed certificates by default:
 - `scp` the certificate over to my main computer and add it as a trusted certificate
 - be pretty sure I'm not doing any of this properly
 - switch back to `http` until I have more time to play around with this

From my point of view, both sides of this disagreement have a point, and the only thing missing is some compromise language. I think it's fair to say the following is true:

- `ankisyncd` does not, technically speaking, support `https`; you need to spin up a reverse proxy to handle the actual SSL and forward bare HTTP to `anksyncd`
- it is incorrect to tell people to avoid `https`; they simply need to be informed of the extra steps required

Given this, may I suggest the README be changed to read as follows:
> Unless you have set up a reverse proxy to handle encrypted connections, use `http` as the protocol. The port will be either the default, 27701, or whatever you have specified in `ankisyncd.conf` (or, if using a reverse proxy, whatever port you configured to accept the front-end connection).
> Do not use trailing slashes.
> Even though the AnkiDroid interface will request an email address, this is not required; it will simply be the username you configured with `ankisyncctl.py adduser`

**Comment by tsudoko** [comment]
Applied @luoliyan's version in 91b022eda62cefbc66d95bf8c48cf8d613dfb961 as it's more correct.

**Pull request closed without merging**